collateral_pools/collateral_pool_1/markets/market_1eth.toml

```diff
@@ -0,0 +1,173 @@
+func = "setFeatureFlagAllowAll"
+args = [
+    "<%= keccak256(defaultAbiCoder.encode(['bytes32','uint128'],[keccak256(hexlify('collateralPoolEnabled')),settings.market1EthId])) %>",
+    true
+]
+depends = [
+  "invoke.upgrade_core_proxy", 
+  "var.market_1eth_vars",
+]
+
+[invoke.market_1eth_set_pausers_cp]
+target = ["CoreProxy"]
+fromCall.func = "getCollateralPoolOwner"
+fromCall.args = ["<%= settings.market1EthId %>"]
+func = "setDeniers"
+args = [
+  "<%= keccak256(defaultAbiCoder.encode(['bytes32','uint128'],[keccak256(hexlify('collateralPoolEnabled')),settings.market1EthId])) %>",
+  [
+    "<%= settings.pauser1 %>",
+    "<%= settings.pauser2 %>",
+    "<%= settings.pauser3 %>",
+    "<%= settings.pauser4 %>",
+  ],
+]
+depends = [
+  "invoke.upgrade_core_proxy", 
+  "var.market_1eth_vars",
+]
+
+##################################################
+# Market Feature Flags
+##################################################
+[invoke.market_1eth_unpause_market]
+target = ["PassivePerpProxy"]
+fromCall.func = "owner"
+func = "setFeatureFlagAllowAll"
+args = [
+    "<%= keccak256(defaultAbiCoder.encode(['bytes32','uint128'],[keccak256(hexlify('marketEnabled')),settings.market1EthId])) %>",
+    true
+]
+depends = [
+  "invoke.upgrade_core_proxy", 
+  "var.market_1eth_vars",
+]
+
+[invoke.market_1eth_set_pausers_market]
+target = ["PassivePerpProxy"]
+fromCall.func = "owner"
+func = "setDeniers"
+args = [
+  "<%= keccak256(defaultAbiCoder.encode(['bytes32','uint128'],[keccak256(hexlify('marketEnabled')),settings.market1EthId])) %>",
+  [
+    "<%= settings.pauser1 %>",
+    "<%= settings.pauser2 %>",
+    "<%= settings.pauser3 %>",
+    "<%= settings.pauser4 %>",
+  ],
+]
+depends = [
+  "invoke.upgrade_core_proxy", 
+  "var.market_1eth_vars",
+]
+
+#####################################
+# Market Configuration
+#####################################
+[invoke.market_1eth_set_market_config]
+target = ["PassivePerpProxy"]
+fromCall.func = "owner"
+func = "setMarketConfiguration"
+args = [
+  "<%= settings.market1EthId %>",
+  { riskMatrixIndex = "<%= settings.market1Eth_riskMatrixIndex %>", maxOpenBase = "<%= parseEther(settings.market1Eth_maxOpenBaseUnscaled) %>", velocityMultiplier = "<%= parseEther(settings.market1Eth_velocityMultiplierUnscaled) %>", oracleNodeId = "<%= settings.ethUsdcNodeId %>", mtmWindow = "<%= settings.market1Eth_mtmWindow %>", dutchConfig = { lambda = "<%= parseEther(settings.market1Eth_dutchConfig_lambdaUnscaled) %>", minBase = "<%= parseEther(settings.market1Eth_dutchConfig_minBaseUnscaled) %>" }, slippageParams = { phi = "<%= parseEther(settings.market1Eth_slippageParams_phiUnscaled) %>", beta = "<%= parseEther(settings.market1Eth_slippageParams_betaUnscaled) %>" }, minimumOrderBase = "<%= parseEther(settings.market1Eth_minimumOrderBaseUnscaled) %>", baseSpacing = "<%= parseEther(settings.market1Eth_baseSpacingUnscaled) %>", priceSpacing = "<%= parseEther(settings.market1Eth_priceSpacingUnscaled) %>"}
```

Comment:
hmm, on this dutch config lambda, can you remind me if we got fully aligned on the values for that with AA? @avniculae 

just to follow up on the above pieces, i think it'd also be really nice to have simulations for dutch liquidations in the fork test as a p0 (ideally followed by immediate closure), we don't need to worry about ae / ranked as these are not needed for day 1

Reply:
i suspect there must be a way to simulate an oracle price jump in the fork test? and then ensure dutch liquidation works as expected but most importantly that the rewards are not out of whack, etc if that makes sense